<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>host_dialog</class>
 <widget class="QDialog" name="host_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Host a game!</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(37, 37, 38);
}
QLabel {
	color: white;
}

QSpinBox {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop: 0 #444444, stop: 1 #222222);
	color: #E0E1E3;
	border-radius: 4px;
	border: 1px;
	border-style: solid;
	border-color: black;
}

QSpinBox:hover {
	border-color: #3DAEFF;
}

QSpinBox::up-button {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QSpinBox */
    border-bottom-right-radius: 3px;
}

QSpinBox::up-arrow {
    image: url(:/resources/spin_up_arrow.png);
    width: 7px;
    height: 7px;
}

QSpinBox::down-button {
    subcontrol-origin: padding;
    subcontrol-position: bottom right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QSpinBox */
    border-bottom-right-radius: 3px;
}

QSpinBox::down-arrow {
    image: url(:/resources/spin_down_arrow.png);
    width: 7px;
    height: 7px;
}

QTabWidget::pane { /* The tab widget frame */
    border-top: 1px solid black;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
	color: white;
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop: 0 #444444, stop: 1 #222222);
    border: 1px solid black;
    border-bottom-color: black; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 4px 8px;
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>82</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="fd1_tab">
    <attribute name="title">
     <string>FNAF 1</string>
    </attribute>
    <widget class="QComboBox" name="fd1_map_cb">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Shelter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Before Midnight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>???</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="fd1_map_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Scene:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fd2_tab">
    <attribute name="title">
     <string>FNAF 2</string>
    </attribute>
    <widget class="QComboBox" name="fd2_map_cb">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Night 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Custom) New and Shiny</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Custom) Double Trouble</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Custom) Foxy Foxy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Custom) Ladies Night</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Custom) Freddy's Circus</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Custom) Balloon Boy is Pissed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(Custom) Golden Freddy</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="fd2_map_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Scene:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fd3_tab">
    <attribute name="title">
     <string>FNAF 3</string>
    </attribute>
    <widget class="QComboBox" name="fd3_map_cb">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Night 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night ???</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night Ed</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="fd3_map_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Scene:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fd4_tab">
    <attribute name="title">
     <string>FNAF 4</string>
    </attribute>
    <widget class="QComboBox" name="fd4_map_cb">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Night 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Night 6</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="fd4_map_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Scene:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>162</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="host_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>162</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="nplayers_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of players:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="nplayers_sb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="netmode_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Netmode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="netmode_cb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>44</y>
     <width>222</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Peer-to-Peer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Packet Server</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>11</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="port_sb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>5029</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nplayers_sb</tabstop>
  <tabstop>port_sb</tabstop>
  <tabstop>netmode_cb</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>fd1_map_cb</tabstop>
  <tabstop>host_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>fd2_map_cb</tabstop>
  <tabstop>fd3_map_cb</tabstop>
  <tabstop>fd4_map_cb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
